src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlow.groovy

```diff
@@ -48,7 +48,7 @@ private void drawBuildInfo(Run build) {
     def colorClasses = color.name().replace('_', ' ') + ' ' + (build == my.target ? 'SELECTED' : '')
     div(class: "build-info ${colorClasses}") {
       a(class: 'build-number model-link inside', href: "${rootURL}/${build.url}") {
-        span("${build.parent.name} ${build.displayName}")
+        span("${build.parent.name} ${build.fullDisplayName}")
```

Comment:
The build.fullDisplayName includes the build.parent.name. So this will look a bit strange. If we where to use build.fullDisplayName, build.parent.name needs to be removed.